### DISCUSSION TITLE
Is the vending machine considered as an embedded device?

### DISCUSSION THREAD

**Discussion by timemaster5** [slint-ui/slint#3987] [Q&A] [answered]
Hi, I am planning to use Slint on a vending machine. Do you know if I can use the Royalty-Free license?

**Comment by ogoffart** [comment, marked as answer]
Yes, a vending machine is an embedded device. So The Royalty-Free license doesn't apply. You can pick another license.

**Reply by timemaster5** [reply]
Thank you !
